pkg/util/ip/ip_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ip
+
+import (
+	"net"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func newCIDR(cidrStr string) *net.IPNet {
+	_, tmpIpNet, _ := net.ParseCIDR(cidrStr)
+	return tmpIpNet
+}
+
+func TestDiffCIDRs(t *testing.T) {
+	testList := []*net.IPNet{newCIDR("10.20.0.0/16"),
+		newCIDR("10.20.1.0/24"),
+		newCIDR("10.20.2.0/28")}
+
+	exceptList1 := []*net.IPNet{testList[1]}
+	correctList1 := []*net.IPNet{newCIDR("10.20.128.0/17"),
+		newCIDR("10.20.64.0/18"),
+		newCIDR("10.20.32.0/19"),
+		newCIDR("10.20.16.0/20"),
+		newCIDR("10.20.8.0/21"),
+		newCIDR("10.20.4.0/22"),
+		newCIDR("10.20.2.0/23"),
+		newCIDR("10.20.0.0/24")}
+
+	diffCIDRs, err := DiffFromCIDRs(testList[0], exceptList1)
+	if err != nil {
+		t.Fatalf("diffFromCIDRs() error = %v", err)
+	} else {
+		assert.Equal(t, correctList1, diffCIDRs)
```

Comment:
you may want to use `assert.ElementsMatch` since the order should not matter for test to pass?

Reply:
same for below occurrences